src/Microsoft.DotNet.Interactive.CSharpProject/CSharpProjectKernel.cs

```diff
@@ -291,9 +291,17 @@ private static IEnumerable<Diagnostic> GetDiagnostics(string code, CompileResult
             projectDiagnostics = projectDiags;
         }
 
-        var allDiagnostics = diagnostics.Concat(projectDiagnostics);
```

Comment:
I wonder if we need the project diagnostics at all. Are there ever diagnostics here that are interesting from the point of view of Try .NET?

Reply:
`Diagnostics `contain errors introduced by the user, while `ProjectDiagnostics `cover errors across the entire project.

Using only `Diagnostics `works well for our primary focus on user-introduced errors.

For debugging purposes and to catch potential errors from changes to the background project, it might be useful to include `ProjectDiagnostics`. However, since the project is pre-built and we assume deployment doesn't occur with build errors, we can safely remove `ProjectDiagnostics `from `GetDiagnostics`.